flocker/node/_change.py

```diff
@@ -110,6 +114,8 @@ def in_parallel(changes):
     Run a series of changes in parallel.
 
     Failures in one change do not prevent other changes from continuing.
+
+    The order in which execution of the changes is started is unspecified.
```

Comment:
But perhaps document the comparability aspect, which I assume is the point of the exercise.